src/FsArchive.cpp

```diff
@@ -19,6 +19,15 @@
 #include "QLZ4.h"
 #include "LZS.h"
 #include "ArchiveObserver.h"
+#include "QRegularExpressionWildcardCompat.h"
+
+#if QT_VERSION < QT_VERSION_CHECK(6, 6, 0)

```

Comment:
why not define this if else into QRegularExpressionWildcardCompat.h ?

Reply:
Done